tests/src/deviceLib/hipTestHalf.cpp

```diff
@@ -19,13 +19,14 @@ THE SOFTWARE.
 
 #include <iostream>
 #include <hip/hip_fp16.h>
-#include "hip/hip_runtime_api.h"
+#include "hip/hip_runtime.h"
+#include "test_common.h"
 
 #define LEN 64
 #define HALF_SIZE 64*sizeof(__half)
 #define HALF2_SIZE 64*sizeof(__half2)
 
-#if __HIP_ARCH_GFX803__ > 0
+#if __HIP_ARCH_GFX803__ || __HIP_ARCH_GFX803__
```

Comment:
Do you mean GFX900 here? Actually the original line i.e. ```_HIP_ARCH_GFX803 > 0``` might still work correctly.